Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/TapToPlace.cs

```diff
@@ -18,6 +17,9 @@ namespace HoloToolkit.Unity.InputModule
     [RequireComponent(typeof(Interpolator))]
     public class TapToPlace : MonoBehaviour, IInputClickHandler
     {
+        [Tooltip("Select which layers you would like to Raycast against.")]
+        public LayerMask RaycastLayerMask = Physics.DefaultRaycastLayers;
```

Comment:
I'm not sure we should be doing this here.  We should let the pointers raycast against certain objects, and the devs set the layer of the objects they're trying to raycast against.